base/linalg/bidiag.jl

```diff
@@ -184,7 +194,7 @@ At_ldiv_B(A::Union{Bidiagonal, AbstractTriangular}, B::AbstractMatrix) = At_ldiv
 function naivesub!{T}(A::Bidiagonal{T}, b::AbstractVector, x::AbstractVector = b)
     N = size(A, 2)
     if N != length(b) || N != length(x)
-        throw(DimensionMismatch())
+        throw(DimensionMismatch("Second dimension of A, $N, does not match one of the lengths of x, $(length(x)), or b, $(length(b))"))
```

Comment:
I think a saw this discussed somewhere recently, but @JeffBezanson stated an error message policy some time ago where errors don't begin with a capital letter. I think this policy is used in the majority of errors in base and hopefully `LinAlg`.